source/documentation/other-topics/persistent-volume-encryption-snapshot.html.md.erb

```diff
@@ -0,0 +1,128 @@
+---
+title: Persistent volume encryption at rest and daily snapshots
+last_reviewed_on: 2019-12-24
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+### Introduction
+
+This guide will give you an overview of how to set encryption at rest for the Persistent volumes and daily snapshots taken to back up Persistent volumes. 
```

Comment:
We need to be consistent with capitalisation. Either "Persistent Volume" or "persistent volume" throughout (except at the start of a sentence). The same applies to other terms like "storage class" - choose a capitalisation scheme and then use it consistently, to avoid confusing the reader.

Also, common practice in technical documentation  is to specify the term plus abbreviation the first time, i.e. "Persistent Volume (PV)" and then optionally to use the abbreviation PV elsewhere in the document.